### PR TITLE
Fix NPE in elasticsearch plugin

### PR DESCRIPTION
**Pull request by xzyJavaX**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking-java/blob/main/CHANGES.md).  
closes [#8969](https://github.com/apache/skywalking/issues/8969)

### CODE REVIEW

**Comment by Superskyyy** [comment]
Please update the Changes log to reflect your fix.

File: apm-sniffer/apm-sdk-plugin/elasticsearch-6.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/elasticsearch/v6/interceptor/AdapterActionFutureActionGetMethodsInterceptor.java [hunk 1]
```diff
@@ -66,7 +67,10 @@ public Object afterMethod(EnhancedInstance objInst, Method method, Object[] allA
     @Override
     public void handleMethodException(EnhancedInstance objInst, Method method, Object[] allArguments,
                                       Class<?>[] argumentsTypes, Throwable t) {
-        ContextManager.activeSpan().log(t);
+        RestClientEnhanceInfo restClientEnhanceInfo = (RestClientEnhanceInfo) objInst.getSkyWalkingDynamicField();
```
Comment (lujiajing1126) [inline apm-sniffer/apm-sdk-plugin/elasticsearch-6.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/elasticsearch/v6/interceptor/AdapterActionFutureActionGetMethodsInterceptor.java:70]: The real type of the dynamic field here is `boolean` as shown in `isTrace()`.

You may reuse `isTrace`. It is much easier.

File: apm-sniffer/apm-sdk-plugin/elasticsearch-7.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/elasticsearch/v7/interceptor/AdapterActionFutureActionGetMethodsInterceptor.java [hunk 2]
```diff
@@ -67,7 +68,10 @@ public Object afterMethod(EnhancedInstance objInst, Method method, Object[] allA
     @Override
     public void handleMethodException(EnhancedInstance objInst, Method method, Object[] allArguments,
                                       Class<?>[] argumentsTypes, Throwable t) {
-        ContextManager.activeSpan().log(t);
+        RestClientEnhanceInfo restClientEnhanceInfo = (RestClientEnhanceInfo) objInst.getSkyWalkingDynamicField();
```
Comment (lujiajing1126) [inline apm-sniffer/apm-sdk-plugin/elasticsearch-7.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/elasticsearch/v7/interceptor/AdapterActionFutureActionGetMethodsInterceptor.java:71]: Same here.

**Review by lujiajing1126 — CHANGES_REQUESTED** [review]

**Comment by wu-sheng** [comment]
Change log is still missing. Please fix.

**Review by lujiajing1126 — APPROVED** [review]
LGTM

**Pull request merged**